src/extension/panel.css

```
@@ -5,6 +5,10 @@
     gap: 12px;
     min-width: 400px;
 }
+
+body {
+    margin: 0px;
+}
 header {
```

Add a new line between 2 css selectors

```suggestion
}

header {
```